app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    for part in command.split(" "):
+        if part[1] != part[2]:
+            with open(part[1], "r") as file_in, open(part[2], "w") as file_out:
```

Comment:
The variables `part[1]` and `part[2]` are incorrectly used here. You should extract the source and destination file names from the command split result and use them appropriately.